cmd/config.go

```diff
@@ -125,20 +125,27 @@ func importCmdCommand() *cobra.Command {
 	var clusterName string
 
 	virtualMachineScaleSetVMs := func() (map[string]*utils.VirtualMachineScaleSetVM, error) {
+		utils.DefaultSpinner.Start()
+		utils.DefaultSpinner.Suffix = " Importing..."
+
 		if subscriptionID != "" && resourceGroup != "" && clusterName != "" {
```

Comment:
nit: I wonder if it still makes sense to do `defer utils.DefaultSpinner.Stop()` here and continue to stop the spinner where explicitly required?

Reply:
If we defer it and there is an error in `VirtualMachineScaleSetVMsViaKubeconfig`. Then, we will fall in the example 1:

```bash
$ ./kubectl-aks config import
/ Importing...WARN[0030] Could not get VMSS VMs via Kubernetes API
WARN[0030] Please provide '--subscription', '--resource-group' and '--cluster-name' flags to get VMSS VMs via Azure API
Error: getting VMSS VMs via Kuberntes API: listing nodes: Get "https://172.28.128.4:6443/api/v1/nodes": dial tcp 172.28.128.4:6443: i/o timeout
```

Reply:
Ah, right! Got it!